Readme.md

```diff
@@ -0,0 +1,181 @@
+# Portfolio
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="Images\html_css.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Abiy Capston One Readme</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ABIY-PORTFOLIO] <a name="about-project"></a>
+
+**[Abiy-Portfolio]** This is a capstone created during my Microverse course. 
+The capstone is able to function (responsive to) both on the mobile and desktop version.
+
+- [Project Description Link](https://www.loom.com/share/4d72f614a9334efc9e8d2958394eafa6)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/Javascript">Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://dart.dev/tools/linter-rules">LINTERS</a></li>
+  </ul>
+</details>
```

Comment:
- Hey, since all the header text for each drop-down is Technologies, you can group them into one or make them a list of items instead of having to duplicate the name.